src/templates/blog-post.js

```diff
@@ -43,7 +43,9 @@ const PostFeaturedImage = ({ post }) => {
   let featuredImageByline
   if (featuredImageCredit) {
     if (featuredImageLink) {
-      featuredImageByline = <a href="{featuredImageLink}">{featuredImageCredit}</a>
+       <a href={featuredImageLink} target="_blank" rel="noopener">
```

Comment:
We want to keep our variable assignment here, so we can use it below.

Reply:
My bad, not sure how I missed that.